model/validation.go

```diff
@@ -838,6 +838,55 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs
 				parentShutdownDuration.Seconds(), drainDuration.Seconds()))
 	}
 
+	if drainDuration > (time.Hour * 1) {
+		errs = multierror.Append(errs,
+			fmt.Errorf("Istio drain time %v must be less than 1 hour", drainDuration.Minutes()))
```

Comment:
[time.Duration](https://golang.org/src/time/time.go#L472) should print the proper units with String().